eva-server/src/test/java/uk/ac/ebi/eva/server/repository/VariantEntityRepositoryTest.java

```diff
@@ -63,7 +63,7 @@
  */
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration
-@UsingDataSet(locations = {"/test-data/variants.json"})
+@UsingDataSet(locations = {"/test-data/eva_hsapiens_grch37.variants_1_2.json"})
```

Comment:
The reason I changed it was to keep better track of the database and collection it had come from. Should I still change it?